ee/tabby-webserver/docs/api_spec.md

```diff
@@ -0,0 +1,169 @@
+# Repositories resolver API
+
+Get file or directory content from local repositories
+
+**URL** : `/repositories/{name}/resolve/{path}`
+
+**Method** : `GET`
+
+**Request examples**
+
+Get directory content
+
+```shell
+curl --request GET \
+  --url http://localhost:8080/repositories/https_github.com_TabbyML_tabby.git/resolve/
+
+curl --request GET \
+  --url http://localhost:9090/repositories/https_github.com_TabbyML_tabby.git/resolve/ee/tabby-webserver/
+```
+
+Get file content
+
+```shell
+curl --request GET \
+  --url http://localhost:8080/repositories/https_github.com_TabbyML_tabby.git/resolve/package.json
+
+curl --request GET \
+  --url http://localhost:9090/repositories/https_github.com_TabbyML_tabby.git/resolve/ee/tabby-webserver/src/api.rs
+```
+
+## Success Responses
```

Comment:
There's no need to split the documentation in such a verbose way (otherwise we should just use openapi doc), provide minimal request / response would be sufficent.